images/virtualization-audit/werf.inc.yaml

```
@@ -0,0 +1,12 @@
+---
+image: {{ $.ImageName }}
+fromImage: distroless
+import:
+  - image: virtualization-artifact
+    add: /usr/local/go/src/virtualization-controller/virtualization-audit
+    to: /app/virtualization-audit
+    after: install
+docker:
+  USER: 64535
+  WORKDIR: "/app"
+  ENTRYPOINT: ["/app/virtualization-audit"]
```

Since we are using werf v2, it should look like this:
```suggestion
imageSpec:
  config:
    user: 64535
    workingDir: "/app"
    entrypoint: ["/app/virtualization-audit"]
```